src/program.js

```diff
@@ -67,6 +70,11 @@ export class Program {
     name: string, description: string, executor: Function,
     commandOptions: Object = {}
   ): Program {
+    this.mapCommandToSubOpts = {};
+    this.mapCommandToSubOpts[name] = commandOptions;
+    this.commandExecuted = name;
+    this.subCommandDefaultValues = setDefaultValues(commandOptions);
```

Comment:
`this.subCommandDefaultValues` is no different from [this.defaultValues](https://github.com/shubheksha/web-ext/blob/854caca3615b512d75d1d88ee850ad9fd9c92a74/src/program.js#L103) except that it has a nested key (the command name). Try to see if you can use `this.defaultValues` for everything -- I think you can.